playground/lib/home/pages/sheet_page_with_dynamic_page_properties.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:demo_ui_components/demo_ui_components.dart';
+import 'package:flutter/material.dart';
+import 'package:playground/home/dynamic_page_properties.dart';
+import 'package:wolt_modal_sheet/wolt_modal_sheet.dart';
+
+class SheetPageWithDynamicPageProperties {
+  SheetPageWithDynamicPageProperties._();
+
```

Comment:
Since this is already being used. Short documentation will be helpful.

Reply:
added docs here: 219fc21b0822d8f94b739f6acbb7f005c66b3c80